### ISSUE TITLE
Multiple --reject/-x only consider the last one

### ISSUE THREAD

**Issue by jwickers** [raineorshine/npm-check-updates#1125]
- [X ] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [X ] I am using the latest version of `npm-check-updates`
- [ X] I am using `node >= 14`

---

## Steps to Reproduce
Add multiple reject patterns like `-x foo -x bar`

## Current Behavior
Multiple --reject/-x only consider the last one, so 
`ncu  -x foo -x bar` only rejects `bar` and ignores `-x foo`

## Expected Behavior
All reject patterns should be considered so
`ncu  -x foo -x bar`
and
`ncu  -x foo,bar`
should be equivalent.

**Comment by raineorshine** [comment]
Fixed and published in `v13.0.4`. Thanks!

**Comment by chemicalkosek** [comment]
Not sure if I should open a new issue, but on latest version 16.7.5 I see the same behavior.

`ncu -x axios -x typeorm`

```bash
[====================] 39/39 100%

 axios                              ^0.27.2  →    ^1.3.3
 ```

**Comment by raineorshine** [comment]
@chemicalkosek Thanks for reporting! That is definitely a regression. I've published a fix to `v16.7.6` and added tests to ensure this doesn't happen again.
